### PR TITLE
Updating feature extraction logic

### PR DESCRIPTION
**Pull request by VMois**
Related issue #7

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 raw-data/*
+data/*
 !raw-data/.gitkeep
 .idea/
+__pycache__/
```
Comment (themreza) [inline .gitignore]: Remove the cache folder.

Comment (themreza) [inline .gitignore]: Remove the cache folder.

File: features_extraction_scripts/ravdess.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        8: 'surprise',
+    }
+    return emotions[label]
+
+
+def ravdess_extract():
+    required_zip_filenames = ['Audio_Speech_Actors_01-24', 'Audio_Speech_Actors_01-24']
+    allowed_emotions = [2, 3, 4, 5, 6]
+
+    if not (os.path.isfile('raw-data/Audio_Speech_Actors_01-24.zip') and os.path.isfile(
```
Comment (themreza) [inline features_extraction_scripts/ravdess.py]: This is hard-coded and defeats the purpose of dynamic file names. Replace the values from required_zip_filenames.

**Comment by themreza** [comment]
Looks good!

File: features_extraction_scripts/ravdess.py (hunk 2 above)
Comment (VMois) [reply features_extraction_scripts/ravdess.py]: Okay, agree. Also, PR is still in WIP, so, please, wait until I will finish. Thanks

Comment (themreza) [reply features_extraction_scripts/ravdess.py]: Just a heads up. If you see something even in WIP, mention it so it's not overlooked.

**Pull request merged**